### ISSUE TITLE
Blackmail and Haas Bioroid: Architect of Tomorrow

### ISSUE THREAD

**Issue by CreativeChaos88** [mtgred/netrunner#2491]
So, had seen an issue yesterday with Blackmail, and have a recommendation for AoT.

Blackmail: Corp installed Blackmail, and the runner removed it from game (Slums). For the remainder of the game, the runner was not receiving prompts for his conspiracy breakers when he encountered the specific subtype. 

AoT: I request the ID be programed when clicked to trigger the ability. Currently, if the runner passes any piece of ICE, rezzed or unrezzed that would be the "first" ICE passed, the corp has to manually rez. By making the ID a clickable ability, it doesn't resolve the issue, but allows the corp to manually do this to reduce time and confusion.

**Comment by JoelCFC25** [comment]
I'm not following this. Blackmail is a Runner card (and an event to boot), so the Corp could never install one. Was this meant to be Blacklist? I'm also not understanding the AoT scenario. Can you lay it out in a specific series of steps?

**Comment by CreativeChaos88** [comment]
Yes, sorry. It was meant to be Blacklist.

As for the AoT, imagine a server 3 unrezzed ICE deep, Heimdall>Ravanna>Eli.
Say you don't rez the Heimdall, but you rez the Ravanna. The system doesn't
prompt you to trigger AoT because the runner passed Heimdall, meeting the
"pass" criteria. Basically, I think the way the code is currently
implemented, it checks to see if the runner passes a bioroid for the
trigger, but if it's unrezzed, it triggers the check, but not the ability,
so the first time you actually pass a rezzed bioroid, it doesn't trigger
the ability. Because of that, I was wondering if there was a way to program
the ID to trigger the ability by clicking it. Does that make more sense?

**Comment by JoelCFC25** [comment]
Thank you for the set of steps. I am not able to reproduce this. 

I stacked 3 bioroid ICE on HQ:  Hudson > Ravana > Zed. All were unrezzed. As Runner, I ran HQ. Corp does not rez outermost Hudson. Runner hits "Continue" to proceed. Corp rezzes middle Ravana. Runner breaks or deducts clicks, whatever. Corp hits "No further action", and when the Runner then hits "Continue", they get a wait prompt that the Corp is using HB:AoT, and the Corp has the "Choose a bioroid to rez" prompt open. Is that not correct?
